packages/config/src/options.ts

```diff
@@ -384,12 +384,12 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     canUpdateDuringTestTime: false,
   }, {
     name: 'viewportHeight',
-    defaultValue: 660,
+    defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? 500 : 660,
     validation: validate.isNumber,
     canUpdateDuringTestTime: true,
   }, {
     name: 'viewportWidth',
-    defaultValue: 1000,
+    defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? 500 : 1000,
```

Comment:
Should this be `options: Cypress.Config`?

Reply:
I'm not sure if it should be `Cypress.Config` because it comes from [here](https://github.com/cypress-io/cypress/blob/10.0-release/packages/server/lib/config.ts#L117) which is when we are building the config in runtime

Reply:
Ok, seems fine then. 👍